modin/pandas/base.py

```diff
@@ -1334,10 +1334,13 @@ def kurt(self, axis=None, skipna=None, level=None, numeric_only=None, **kwargs):
                 query_compiler=self._query_compiler.apply("kurt", axis, **func_kwargs)
             )
 
-        if numeric_only:
+        if numeric_only is not None and not numeric_only:
             self._validate_dtypes(numeric_only=True)
```

Comment:
Why condition include `not numeric_only` statement, but `_validate_dtypes` used `numeric_only=True`?

Reply:
if in the reduction function we're getting `numeric_only=False` then we should raise an exception if the frame contains non-numeric data. And that's what `_validate_dtypes(numeric_only=True)` do, it raises an exception if frame contains non-numeric column

Reply:
Got it